cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -103,6 +105,10 @@
 import javax.xml.bind.JAXBContext;
 import javax.xml.bind.JAXBException;
 import javax.xml.bind.Unmarshaller;
+import javax.xml.ws.Response;
+
+import jdk.nashorn.internal.runtime.Version;
```

Comment:
Make sure all the unused dependencies are stripped out, I don't expect the `Java.xml.ws` or `jdk.nashorn` packages to get added here

Reply:
done